Packs/Anomali_Enterprise/Integrations/Anomali_Enterprise/Anomali_Enterprise.py

```diff
@@ -0,0 +1,290 @@
+from typing import Dict, Callable
+
+import urllib3
+from CommonServerPython import *
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+VENDOR_NAME = 'Anomali Enterprise'
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """
+    Client to use in the Anomali Enterprise integration. Overrides BaseClient
+    """
+
+    def __init__(self, server_url: str, username: str, password: str, verify: bool, proxy: bool):
+        super().__init__(base_url=server_url, verify=verify, proxy=proxy)
+        self._username = username
+        self._password = password
+        self._headers = {'Content-Type': 'application/json'}
+
+    def start_search_job_request(self, from_: str, to_: str, indicators: List[str]) -> dict:
+        """Initiate a search job.
+        Args:
+            from_: from which time to initiate the search
+            to_: to which time to initiate the search
+            indicators: indicators to search
+        Returns:
+            Response from API.
+        """
+        data = json.dumps({'username': self._username, 'password': self._password, 'from': from_, 'to': to_,
```

Comment:
you can skip the `json.dumps` if you use the argument `json_data` instead. This is true for subsequent uses of `BaseClient._http_request()`, so I won't repeat this comment.